### PR TITLE
[WIP][IS-1505  and IS-1506] Add integration tests

### PR DESCRIPTION
**Pull request by anikitinDSR**

### CODE REVIEW

File: libindy_vdr/src/ledger/request_builder.rs [hunk 1]
```diff
@@ -711,17 +693,17 @@ impl RequestBuilder {
...
+        rich_schema: RichSchema,
     ) -> VdrResult<PreparedRequest> {
-        let rs_schema: RichSchema = RichSchema::new(
-            rs_schema.id,
-            rs_schema.content,
-            rs_schema.rs_name,
-            rs_schema.rs_version,
-            rs_schema.rs_type,
-            rs_schema.ver,
+        let rich_schema: RichSchema = RichSchema::new(
```
Comment (jovfer) [inline libindy_vdr/src/ledger/request_builder.rs]: Should `Clone` trait be derived/implemented for RichSchema struct?

File: libindy_vdr/src/ledger/requests/rich_schema.rs [hunk 2]
```diff
@@ -81,10 +79,13 @@ pub struct RichSchemaOperation {
     pub _type: String,
     pub id: RichSchemaId,
     pub content: RSContent,
+    #[serde(rename = "rsName")]
```
Comment (jovfer) [inline libindy_vdr/src/ledger/requests/rich_schema.rs]: `#[serde(rename_all = "camelCase")]` may be applied to whole sturcture

File: libindy_vdr/src/ledger/request_builder.rs (hunk 1 above)
Comment (anikitinDSR) [reply libindy_vdr/src/ledger/request_builder.rs]: i will delete RichSchema object recreation at all

**Comment by andrewwhitehead** [comment]
@anikitinDSR @Artemkaaas  It sounds like this won't be actively worked on for a bit. I'd like to merge this so that I can clean up any conflicts with my PR, and then add a feature flag around the rich schemas parts for now. Any objections?

**Pull request merged**